app/Inventory.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+namespace App;
+use App\Helper\DataViewer;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Inventory extends Model
+{
+    //
+    use DataViewer;
+
+
+    protected $fillable = ['receiver_id','branch_id', 'supplier_id','product_id', 'inventory_sku', 'serial_number', 'market_price', 'sold_date', 'received_date', 'status'];
+
+    public static $columns = ['id', 'inventory sku', 'serial_number', 'market_price', 'branch', 'received by' ];
+
+    public static function form(): iterable
```

Comment:
what does this do?

Reply:
Inventory is a repo for products that are availablefor sale at show rooms hence the branch field